docs/Makefile

```diff
@@ -9,16 +9,17 @@ datadir := $(datarootdir)
 mandir := $(datarootdir)/man
 bindir :=  $(PREFIX)/bin
 
-all: quickget.1 quickemu.1 quickemu_conf.1
+all: quickget.1 quickemu.1 quickemu_conf.5
 
 clean:
-	rm *.1
+	rm *.1 *.5
 
 install_docs: all
 	install -d $(DESTDIR)$(mandir)/man1
+	install -d $(DESTDIR)$(mandir)/man5
 	install -m 644 quickget.1 $(DESTDIR)$(mandir)/man1
 	install -m 644 quickemu.1 $(DESTDIR)$(mandir)/man1
-	install -m 644 quickemu_conf.1 $(DESTDIR)$(mandir)/man1
+	install -m 644 quickemu_conf.5 $(DESTDIR)$(mandir)/man5
 
```

Comment:
```suggestion
    rm -f $(DESTDIR)$(mandir)/man1/quickemu_conf.1
```

Reply:
Good point, I pushed https://github.com/quickemu-project/quickemu/pull/1454/commits/27bdac91cf7eb6d58d770d30fafd537d3dec2d43 to the pull request's branch.